librabbitmq/amqp_cfstream_socket.h

```diff
@@ -0,0 +1,30 @@
+//
+//  amqp_cfstream_socket.h
+//  AMQPKit
```

Comment:
If this is going to be a part of rabbitmq-c it must have the rabbitmq-c license. Please make the headers similar to what is in amqp.h (though you'll want to assign the copyright to yourself if you've created the file).